tg/views.py

```diff
@@ -14,7 +14,6 @@
 
 MAX_KEYBINDING_LENGTH = 5
 MULTICHAR_KEYBINDINGS = (
-    "gg",
```

Comment:
why was it deleted? I like to have `gg` shortcut. we need to take it back

Reply:
I've just thought it would be better to use g instead of gg (less uses g but vim uses gg).
Which one would you prefer?